src/components/blocks/layout/Hero.tsx

```diff
@@ -0,0 +1,78 @@
+import { Image } from '@mantine/core'
+import { Template } from 'tinacms'
+import { PageBlocksHero } from '~tina/__generated__/types'
+import { TinaMarkdown } from 'tinacms/dist/rich-text'
+import {
+	tinaMarkdownComponentsRichTextTemplate,
+	tinaMarkdownComponents,
+} from '../../../components/fields/TinaMarkdownComponents/TinaMarkdownComponents'
+import { cn } from '../../../lib/utils'
+
+export const Hero = ({ data }: { data: PageBlocksHero }) => {
+	return (
+		<>
+			{data.heroBackgroundImage?.src && data.heroBackgroundImage?.alt && (
+				<Image
+					src={data.heroBackgroundImage.src}
+					alt={data.heroBackgroundImage.alt}
+					className='md:h-[480px] h-[300px]'
+				/>
+			)}
+			<div
+				className={cn('absolute prose top-1/2 transform -translate-y-1/2 px-6 md:prose-h1:text-6xl prose-h1:text-4xl max-w-none flex flex-col gap-4', {
```

Comment:
you could just have a div with the chosen background image, and the title is centered using flex-box rather than absolute